rustc-ice-2024-08-19T10_33_10-40406.txt

```diff
@@ -0,0 +1,599 @@
+thread 'rustc' panicked at compiler/rustc_middle/src/ty/normalize_erasing_regions.rs:187:90:
```

Comment:
Are you including this file in PR?

Reply:
That one sneaked in probably. Need to remove it.